app/scripts/controllers/containersgrid.js

```diff
@@ -50,12 +50,12 @@ angular.module('dockstore.ui')
       $scope.hasDescription = function(description) {
         // temporary
         return false;
-        //
-        if (description !== undefined && description !== null && description !== '' && $scope.homePage === false) {
-          return 'search-with-description';
-        } else {
-          return '';
-        }
+        // Don't delete
```

Comment:
Maybe add a reason so that it will become more clear to future developers why this was added/retained